### DISCUSSION TITLE
Auto Create Column

### DISCUSSION THREAD

**Discussion by ceicomcloud** [Power-Components/livewire-powergrid#417] [Q&A] [answered]
Hi,

Have you an idea for auto create column based on the data ?
EX:

  ['id' => 1, 'name' => 'Name 1', 'price' => 1.58, 'created_at' => now()

i want to automatically create name, price and created_at without have to manually put the addColumn or Column::add

Thanks !

**Comment by CrazyBoy49z** [comment, marked as answer]
This feature has already been created

https://user-images.githubusercontent.com/3448603/168392446-47825404-3fe7-4023-ad75-03879d801bad.mov

**Comment by luanfreitasdev** [comment]
Thanks for this Video!.

**Comment by CrazyBoy49z** [comment]
I wanted to download better videos, but GitHub doesn't allow more than 10 MB
